python/tank/descriptor/io_descriptor/base.py

```diff
@@ -554,7 +565,47 @@ def exists_local(self):
         """
         Returns true if this item exists in a locally accessible form
         """
-        return self.get_path() is not None
+        # first get the path
+        path = self.get_path()
+        return self._exists_local(path)
+
+    def _exists_local(self, path):
+        """
+        Returns true if the given bundle path exists in valid local cached form
+        """
+        if not path:
+            # not on disk!
+            return False
+
+        # check that the main path exists locally and is a folder
+        if not os.path.isdir(path):
+            return False
```

Comment:
This comment should be ignored. Its from a previous PR. It doesn't apply anymore.